Flabbergasted-Flamingos/bangazon-commerce-app
# Rb product detail

# Description
You can now add products to shopping cart AND the product quantity decreases 🥇 . Also added a hyperlink and grumpy face to logo so that you can click on it to view list of products

## Number of Fixes
Finally finished #10 


## Problem to Solve
You can add products now to shopping cart

## Proposed Changes
Affects urls, views, templates and navbar

## Expected Behavior
You can add to the cart from the detail product view and it's magnificent

## Steps to Test Solution

1. 
```
git fetch --all
git checkout rb_product_detail
python manage.py runserver
```
Go to shopping cart and see that you have atleast one item in there, then click on the grumpy face to go back to list of products.

Once you have all the products click on the hyperlink for one so you can view the detail.

Click the add to cart button and then see that indeed, you have two items in your cart. 

Go back to the product detail for what you just added and make sure that the quantity is decreased.

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation
[x ] There is new documentation in this pull request that must be reviewed..
I added the sql queries to the read me


## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.
